etc/docker/demo-base/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG BASE_CONTAINER=ubuntu:bionic
+ARG BASE_CONTAINER=continuumio/miniconda3
```

Comment:
I am assuming you are doing this to simplify the removal of miniconda install which is platform-specific (`Miniconda3-py37_4.8.2-Linux-x86_64.sh`) ?

Reply:
> @leoyifeng assuming today we usually do `make docker-images publish-docker-images` how does this pr change this user experience? With this, we would need to do `MULTIARCH=true make docker-images` assuming the multi-arch pushes directly to the docker repository.

yes, with my current implementation, user has to specify the MULTIARCH parameter to trigger the buildx build. Do we want to build multi arch images by default going forward ?

Reply:
> I am assuming you are doing this to simplify the removal of miniconda install which is platform-specific (`Miniconda3-py37_4.8.2-Linux-x86_64.sh`) ?

yes